internal/provider/data_source_image_repo.go

```diff
@@ -208,7 +208,7 @@ func (d *imageRepoDataSource) Read(ctx context.Context, req datasource.ReadReque
 				Source:      types.StringValue(repo.GetSyncConfig().GetSource()),
 				Expiration:  types.StringValue(repo.GetSyncConfig().GetExpiration().AsTime().Format(time.RFC3339)),
 				UniqueTags:  types.BoolValue(repo.GetSyncConfig().GetUniqueTags()),
-				SyncAPKs:    types.BoolValue(repo.GetSyncConfig().GetSyncApks()),
+				SyncAPKs:    types.BoolValue(false), // deprecated field, always return false
```

Comment:
Either remove this entirely or default it to true. We always set this to true in the API response (for now, until we fully remove it), so I want to avoid constantly creating diffs.

There shouldn't be a need to set it, but we should double check we aren't generating diffs on the client side that will trick terraform into always wanting to apply a change.